pkg/service/redisstore.go

```diff
@@ -33,9 +33,9 @@ const (
 
 	// IngressKey is a hash of ingressID => ingress info
 	IngressKey         = "ingress"
-	StreamKeyKey       = "stream_key"
-	IngressStatePrefix = "ingress_state:"
-	RoomIngressPrefix  = "room_ingress:"
+	StreamKeyKey       = "{ingress}_stream_key"
+	IngressStatePrefix = "{ingress}_state:"
+	RoomIngressPrefix  = "room_{ingress}:"
```

Comment:
so `{key}` tells RedisCluster where to place the item? TIL

Reply:
It indicates what part of the key to hash to determine the slot.